src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1507,6 +1507,12 @@
           "type": "string"
         },
 
+        "extensionIndex": {
+          "description": "The index within the run.tool.extensions array of the tool component object which describes the plug-in or tool extension that produced the result.",
+          "type": "integer",
+          "minimum": -1
```

Comment:
>minimum [](start = 11, length = 7)

and: `"default": -1`

Reply:
Yes, did some clean up for minimum/default, please take a look at update.

---
In reply to: [255688316](https://github.com/Microsoft/sarif-sdk/pull/1277#discussion_r255688316) [](ancestors = 255688316)